.registryrc

```diff
@@ -1 +1 @@
-8583d0841615313c8c880e765eba760378e061cd
+1140439a0ef6aff514e0db9f9b69d5f07dc3e2c3
```

Comment:
update after https://github.com/hyperlane-xyz/hyperlane-registry/pull/310 is merged